__tests__/lib/openapi-to-json-schema.test.ts

```diff
@@ -703,7 +703,11 @@ describe('`example` / `examples` support', () => {
       }
 
       return {
-        distinctName: {
+        // Since this doesn't have a `value` that we can use we should ignore it.
+        distinctExternal: {
+          externalValue: 'https://example.com/example.example',
+        },
```

Comment:
I didn't change any assertions or create any new explicit tests here because by putting this above `distinctExample` since it doesn't have a `value` all of the tests within this `describe` block are now guaranteed to fallback to using `distinctExample`.